site/en/tutorials/images/transfer_learning.ipynb

```diff
@@ -694,7 +696,6 @@
       },
       "outputs": [],
       "source": [
-        "initial_epochs = 10\n",
```

Comment:
This change may have introduced 1 of 2 issues that broke the notebook (usually, it's because of the end of line `\n`s in JSON). I will fix this in a separate commit by reverting this change. Please edit using Colab/Jupyter in future.